src/net_processing.cpp

```diff
@@ -2406,11 +2406,13 @@ void PeerManagerImpl::RelayInvFiltered(const CInv& inv, const CTransaction& rela
 {
     // TODO: Migrate to iteration through m_peer_map
     m_connman.ForEachNode([&](CNode* pnode) {
+        if (!pnode->CanRelay()) return;
```

Comment:
Consider TODO above, if you have touched this code

Reply:
At some point, I may try to, but it's non-trivial